galvan/src/main/java/org/terracotta/testing/master/GalvanStateInterlock.java

```diff
@@ -59,11 +60,25 @@ public GalvanStateInterlock(ContextualLogger logger, ITestWaiter sharedLockState
     this.sharedLockState = sharedLockState;
   }
 
+  public void ignoreServerCrashes() {
+    synchronized (this.sharedLockState) {
+      ignoreServerCrashes = true;
+      if (activeServer != null) {
+        activeServer.setCrashExpected();
+      }
+      passiveServers.forEach(process->process.setCrashExpected());
+      unknownRunningServers.forEach(process->process.setCrashExpected());
+      terminatedServers.forEach(process->process.setCrashExpected());
```

Comment:
what if we call `ignoreServerCrashes()` in our test, but the server does not crash ?
(I don't understand the method names perhaps: ignore is calling crash expected :wink: )